README.md

```diff
@@ -1899,13 +1899,124 @@ and verify your domain. Generate an API key for the Mailgun account. Note your M
 
 #### Configure Mailgun
 
-Configure the Mailgun credentials in your .env file. Add your Mailgun key to the application.yaml as
-seen below:
+To configure Mailgun with the proper credentials an engineer must pass a mailgun domain,
+mailgun key, and a mailgun sender-email into their application. Those keys are passed into
+the application.yaml as displayed below:
 
 ```yaml
   email-client:
     mailgun:
-      key: ${MAILGUN_KEY}
+      key: ${MAILGUN_KEY:'no-key-set'}
+      domain: 'mail.forms-starter.cfa-platforms.org'
+      sender-email: 'UBI Demo <demo@mail.forms-starter.cfa-platforms.org>'
+```
+
+##### Required Mailgun keys:
+
+| Name           | Descriptor       | How you get it                                                   | 
+|----------------|------------------|------------------------------------------------------------------|
+| `key`          | `MAILGUN_KEY`    | passed in through .env files or secrets.                         |
+| `domain`       | `Mailgun Domain` | passed into application.yaml as a string into the domain field.  |
+| `sender-email` | `Sender Email`   | passed into the application.yaml file in the sender-email field. | 
+
+##### Required Fields for Emails
+
+To build an email message you must pass at least three String arguments into the sendEmail()
+method a MailgunEmailClient object:
```

Comment:
Should this method **on a** MailgunEmailClient ?

Reply:
fixed